lib/types.js

```diff
@@ -28,149 +36,146 @@ export type Pagination = {|
+/**
  */
 export type ChatOptionsRead = {|
   channel: ChatChannel,
-  conversation_id: string,
+  conversationId?: string,
+  peek: boolean,
   pagination?: Pagination,
-  unread_only: boolean,
-  fail_offline: boolean,
+  unreadOnly: boolean,
+  failOffline: boolean,
 |}
 
-/*
- * Read
- * {"method": "edit", "params": {"options": { ...ChatOptionsEdit }}}
+/**
  */
 export type ChatOptionsEdit = {|
   channel: ChatChannel,
-  conversation_id: string,
-  message_id: number,
+  conversationId?: string,
+  messageId: number,
   message: ChatMessage,
 |}
 
-/*
- * Reaction
- * {"method": "reaction", "params": {"options": { ...ChatOptionsReadtion }}}
+/**
  */
 export type ChatOptionsReaction = {|
   channel: ChatChannel,
-  conversation_id: string,
-  message_id: number,
+  conversationId?: string,
+  messageId: number,
   message: ChatMessage,
 |}
 
-/*
- * Delete
- * {"method": "delete", "params": {"options": { ...ChatOptionDelete }}}
+/**
  */
 export type ChatOptionsDelete = {|
   channel: ChatChannel,
-  conversation_id: string,
-  message_id: number,
+  conversationId?: string,
+  messageId: number,
 |}
 
-/*
- * Attachment
- * {"method": "attachment", "params": {"options": { ...ChatOptionAttachment }}}
+/**
  */
 export type ChatOptionsAttachment = {|
   channel: ChatChannel,
-  conversation_id: string,
+  conversationId?: string,
   filename: string,
   title: string,
   preview?: string,
 |}
 
-/*
- * Download
- * {"method": "download", "params": {"options": { ...ChatOptionDownload }}}
+/**
  */
 export type ChatOptionsDownload = {|
   channel: ChatChannel,
-  conversation_id: string,
-  message_id: number,
+  conversationId?: string,
+  messageId: number,
   output: string,
   preview?: string,
-  no_stream?: boolean,
+  noStream?: boolean,
 |}
 
-/*
- * SetStatus
- * {"method": "setStatus", "params": {"options": { ...ChatOptionSetStatus }}}
+/**
  */
 export type ChatOptionsSetStatus = {|
   channel: ChatChannel,
-  conversation_id: string,
+  conversationId?: string,
   status: string,
 |}
 
-/*
- * Mark
- * {"method": "mark", "params": {"options": { ...ChatOptionMark }}}
+/**
  */
 export type ChatOptionsMark = {|
   channel: ChatChannel,
-  conversation_id: string,
-  message_id: number,
+  conversationId?: string,
+  messageId: number,
 |}
 
-/*
- * SearchInbox
- * {"method": "searchInbox", "params": {"options": { ...ChatOptionsSearchInbox }}}
+/**
  */
 export type ChatOptionsSearchInbox = ChatOptionsSearch & {|
   query: string,
-  force_reindex?: boolean,
+  forceReindex?: boolean,
 |}
 
-/*
- * SearchRegexp
- * {"method": "searchregexp", "params": {"options": { ...ChatOptionsSearchRegexp }}}
+/**
  */
 export type ChatOptionsSearchRegexp = ChatOptionsSearchInbox & {|
   channel: ChatChannel,
-  conversation_id: string,
+  conversationId?: string,
   query: string,
-  is_regex: boolean,
-  max_messages: number,
+  isRegex: boolean,
+  maxMessages: number,
 |}
 
+/**
```

Comment:
Are these empty comments so documentation.js notices the types?

Reply:
Yes!